shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -51,26 +61,36 @@ public String getCommandName() {
   }
 
   @Override
+  public Options getOptions() {
+    return new Options()
+        .addOption(LOCAL_OPTION);
+  }
+
+  @Override
   protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
-    load(path);
+    load(path, cl.hasOption(LOCAL_OPTION.getLongOpt()));
   }
 
   /**
-   * Loads a file or directory in Alluxio space, makes it resident in memory.
+   * Loads a file or directory in Alluxio space, makes it resident in Alluxio.
    *
-   * @param filePath The {@link AlluxioURI} path to load into Alluxio memory
+   * @param filePath The {@link AlluxioURI} path to load into Alluxio
+   * @param local the local flag; If the file is already in Alluxio fully, enable the local flag,
+   *              Alluxio will still load the file, otherwise Alluxio will do nothing.
    */
-  private void load(AlluxioURI filePath) throws AlluxioException, IOException {
+  private void load(AlluxioURI filePath, boolean local) throws AlluxioException, IOException {
     URIStatus status = mFileSystem.getStatus(filePath);
     if (status.isFolder()) {
       List<URIStatus> statuses = mFileSystem.listStatus(filePath);
       for (URIStatus uriStatus : statuses) {
         AlluxioURI newPath = new AlluxioURI(uriStatus.getPath());
-        load(newPath);
+        load(newPath, local);
       }
     } else {
-      if (status.getInAlluxioPercentage() == 100) {
-        // The file has already been fully loaded into Alluxio memory.
+      if (!(local && FileSystemContext.INSTANCE.hasLocalWorker())
```

Comment:
if the user sets `--local` but there is no local worker available, they should get an error message

Reply:
Done.